connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoCsvWriter.scala

```diff
@@ -0,0 +1,52 @@
+package com.microsoft.kusto.spark.datasink
+
+import java.io.Writer
+
+case class KustoCsvWriter(out: Writer) {
+  val newLineSep: String = java.security.AccessController.doPrivileged(
+    new sun.security.action.GetPropertyAction("line.separator"))
+  val newLineSepLength: Int = newLineSep.length
+  var progress: Long = 0L
```

Comment:
progress usually is in terms of 0 to 100, or in other words, progress implies it's number is meaningful for progression tracking.. here you are just counting bytes..so I would rename to `bytsCounter` or something similar